src/generators/activityContentGenerators.ts

```diff
@@ -0,0 +1,81 @@
+import { sample, sampleText } from "@dsnp/test-generators";
+
+import {
```

Comment:
what are the difference between a generator and a factory?

Reply:
I think, within our repo at least, "factories" are just general factory functions which help the user create objects more easily and "generators" are functions specifically for creating fake objects for testing. "Factories" ask the user for all their data to go in the object except for what they can infer from types. "Generators" make up fake data to use.

Reply:
That said, these terms are totally overloaded and messy, and I would be totally up for a rename.

Reply:
oohhh i see this was stuff from before. its ok.